src/Movies.js

```diff
@@ -0,0 +1,24 @@
+import './Movies.css';
+import Card from './Card';
+
+function Movies({ movies }){
```

Comment:
do we want ES5 functions or ES6 functions? (function keyword or arrow functions)?

Reply:
ES6 sounds good to me, actually. I made the changes and pushed!